docs/product-assembly_designing-and-building-cases/src/main/asciidoc/case-management-closing-cases-ref.adoc

```diff
@@ -0,0 +1,5 @@
+[id='case-management-closing-cases-ref']
+= Closing cases 
```

Comment:
Modules like this are not strictly "allowed" in our mod doc approach. The only files that should have includes are assemblies, and even those need intro statements/paragraphs. I'd make this module a con, move the intro you're currently using for "Closing a case" to this (fits perfectly, imo), and then move these includes to the assembly, following the include for this actual module. This is more in line with the "protocol".

Here's the intro I recommend for this module, roughly what you already have from "Closing a case":

**A case instance can be completed when there are no more activities to be performed and the business goal was achieved, or it can be closed prematurely. Usually the case owner closes the case when all work is completed and the case goals have been met. When you close a case, consider adding a comment about why the case instance is being closed, especially if it is closed before the business goal was achieved. A closed case can be reopened later with the same case ID if required.**

**You can close case instances remotely using {KIE_SERVER} REST API requests or directly in the Showcase application.**


Also, technically since this is a con, it shouldn't be a gerund anyway. Frankly, I do this occasionally too whenever I have a "Doing something" super procedure that I use as a con because the sub-procedures are really the procedures. I know that Christy doesn't like that and she would recommend **= Closed cases** or something here, for example. I personally don't have a problem with gerunded cons in cases like this. I do see her point, but it just seems less clear to me. 

Up to you, but either way, should be it's own module with its own intro, with no includes. Unless of course you want to make this a separate assembly, but I don't think that's what you want here.